src/main/kotlin/com/autonomousapps/tasks/PrintDominatorTreeTask.kt

```diff
@@ -21,6 +21,6 @@ abstract class PrintDominatorTreeTask : DefaultTask() {
   abstract val consoleText: RegularFileProperty
 
   @TaskAction fun action() {
-    logger.quiet(consoleText.readText())
+    consoleText.readLines().forEach { logger.quiet(it) }
```

Comment:
Why this change?

Reply:
The `readText()` method loads the entire file in memory at once which was proven to be problematic in other places. Refactored it like this as it's functionally the same.

Reply:
Can you share the output here as an example so that I can see what it looks like?

Reply:
After giving it a bit more thought it's slightly different, the file will be printed but now line by line which means a task running in parallel could print other stuff in between which is undesired. Will revert this particular line.